server/file.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const remark = require('remark');
+const lint = require('remark-lint');
+const html = require('remark-html');
+
+function getMarkdown(fileName) {
+  return new Promise((resolve, reject) => {
+    fs.readFile(fileName, (err, markdown) => {
+      if (!err) {
+        resolve(markdown.toString('utf-8'));
+      } else {
+        reject(err.message);
+      }
+    });
+  });
+}
+
+function getFileContent() {
+  return new Promise((resolve, reject) => {
+    getMarkdown('./TEST.md').then((markdown) => {
+      remark().use(lint).use(html).process(markdown, function (err, markdownHtml) {
```

Comment:
about `lint`. I have no idea:) Just used this example - https://github.com/wooorm/remark/tree/master/packages/remark#usage